### PR TITLE
[RW-9642][risk=no]   Warn Users that Jobs may be Terminated when Deleting Cromwell App

### PR DESCRIPTION
**Pull request by NehaBroad**
https://user-images.githubusercontent.com/34481816/226350384-5702b10f-65d2-4cb3-8241-6e9d10677d41.mov

Updated screenshot after Joy's comment
Screenshot 2023-03-21 at 11 27 02 AM

Description:

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have added explanatory comments where the logic is not obvious
- [x] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [x] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: ui/src/app/components/apps-panel/delete-cromwell-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+});
+
+export const DeleteCromwellConfirmationModal = (props: {
+  clickYes;
+  clickNo;
+}) => {
+  let toast: any;
+  let toastTimer: NodeJS.Timer;
+
+  const cromshell_job_check_command = 'cromshell-alpha list -u';
```
Comment (jmthibault79) [inline ui/src/app/components/apps-panel/delete-cromwell-modal.tsx:51]: Screenshot 2023-03-21 at 2 16 45 PM

**Review by jmthibault79 — CHANGES_REQUESTED** [review]
Command given does not work.

File: ui/src/app/components/apps-panel/delete-cromwell-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                <FontAwesomeIcon
+                  size={'xl'}
+                  icon={faClipboard}
+                  style={styles.copyIcon}
+                />
+              </TooltipTrigger>
+            </Clickable>
+            <Toast ref={(el) => (toast = el)} style={styles.toast} />
+          </FlexRow>
+          Do you still want to delete Cromwell?
```
Comment (jmthibault79) [inline ui/src/app/components/apps-panel/delete-cromwell-modal.tsx:94]: "your Cromwell environment"

**Comment by NehaBroad** [comment]
shoot i was trying some command and it merged it.

**Pull request closed without merging**